airbyte-integrations/connectors/destination-snowflake-cortex/integration_tests/sample_config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "processing": {
+    "text_fields": ["str_col"],
+    "metadata_fields": ["int_col"],
+    "chunk_size": 1000
+  },
+  "embedding": {
+    "mode": "openai",
+    "openai_key": "mykey"
+  },
+  "indexing": {
+    "host": "MYACCOUNT",
+    "role": "MYUSERNAME",
+    "warehouse": "COMPUTE_WH",
+    "database": "MYDATABASE",
+    "default_schema": "MYSCHEMA",
+    "username": "MYUSERNAME",
+    "password": "xxxxxxx"
```

Comment:
Should `password` be nested under `credentials` now?

Weird that this didn't throw an error.

Reply:
good catch.. this file is for reference purpose, doesn't get used :) i will update it.